### PR TITLE
Adds keymaps for muzfuz DZ60, Planck, Clueboard66

### PR DESCRIPTION
**Pull request by muzfuz**
Adds keymaps for Planck, DZ60 and Clueboard66
👍

### CODE REVIEW

File: keyboards/clueboard/66/keymaps/muzfuz/keymap.c [hunk 1]
```diff
@@ -0,0 +1,70 @@
+#include "66.h"
+
+// Helpful defines
+#define ESC_CTL     CTL_T(KC_ESCAPE)            // Tap for Esc, hold for Ctrl
+#define _______ KC_TRNS
```
[the remark this answers is not in the document]
Comment (drashna) [reply keyboards/clueboard/66/keymaps/muzfuz/keymap.c]: These defines aren't necessary, as these are actually defined in the core.  You can safely remove them. 

https://github.com/qmk/qmk_firmware/blob/a0309db983150fd0197eb620a6ba552d90d29f93/quantum/quantum_keycodes.h#L30-L31


```suggestion
```

File: keyboards/clueboard/66/keymaps/muzfuz/keymap.c [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+   */
+[_CL] = KEYMAP(
+  BL_STEP,RGB_M_P,RGB_M_B,RGB_M_R,RGB_M_SW,RGB_M_SN,RGB_M_K,RGB_M_X,RGB_M_G,_______,_______,_______,_______,_______,RGB_TOG,        RGB_VAI, \
+  _______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,RESET,                    RGB_VAD, \
+  _______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,                         \
+  _______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,        RGB_SAI,         \
+  _______,_______,_______,_______,        RGB_MOD,   RGB_MOD,                     _______,_______,_______,_______,RGB_HUD,RGB_SAD,RGB_HUI),
+};
+
+const macro_t *action_get_macro(keyrecord_t *record, uint8_t id, uint8_t opt) {
```
Comment (drashna) [inline keyboards/clueboard/66/keymaps/muzfuz/keymap.c]: Could you remove the `action_get_macro` function?  It's been deprecated in favor of `process_record_user`. 

https://docs.qmk.fm/#/feature_macros

File: keyboards/clueboard/66/keymaps/muzfuz/rules.mk [hunk 3]
```diff
@@ -0,0 +1 @@
+DEFAULT_FOLDER = clueboard/66/rev3
```
Comment (drashna) [inline keyboards/clueboard/66/keymaps/muzfuz/rules.mk]: This isn't actually needed. 
```suggestion
```

File: keyboards/dz60/keymaps/muzfuz/keymap.c [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+
+	if (usb_led & (1 << USB_LED_KANA)) {
+
+	} else {
+
+	}
+
+}
+
+const macro_t *action_get_macro(keyrecord_t *record, uint8_t id, uint8_t opt) {
```
Comment (drashna) [inline keyboards/dz60/keymaps/muzfuz/keymap.c]: Could you remove the `action_get_macro` function?  It's been deprecated in favor of `process_record_user`.

File: keyboards/planck/keymaps/muzfuz/keymap.c [hunk 5]
```diff
@@ -0,0 +1,404 @@
...
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include QMK_KEYBOARD_H
+#include "muse.h"
+
+extern keymap_config_t keymap_config;
+
+enum planck_layers
```
Comment (drashna) [inline keyboards/planck/keymaps/muzfuz/keymap.c]: This is optional, and mor of a nitpick (so you can ignore).

Would you be willing to reformat this so it's conforming to the QMK Coding conventions? 

https://docs.qmk.fm/#/contributing?id=coding-conventions

File: keyboards/planck/keymaps/muzfuz/keymap.c [hunk 6]
```diff
@@ -0,0 +1,404 @@
...
+  {
+  case RAISE:
+  case LOWER:
+    return false;
+  default:
+    return true;
+  }
+}
+
+const macro_t *action_get_macro(keyrecord_t *record, uint8_t id, uint8_t opt)
```
Comment (drashna) [inline keyboards/planck/keymaps/muzfuz/keymap.c]: Could you remove the `action_get_macro` function?  It's been deprecated in favor of `process_record_user`.

**Comment by muzfuz** [comment]
@drashna @mechmerlin thank you for the reviews. 😍  I have made appropriate changes.

The only one I didn't change is the formatting request in the Planck keymap, as this is in line with the default template at the moment.

**Comment by drashna** [comment]
Thanks!

**Pull request merged**